openmdao/core/total_jac.py

```diff
@@ -1454,6 +1454,11 @@ def compute_totals_approx(self, initialize=False):
         initialize : bool
             Set to True to re-initialize the FD in model. This is only needed when manually
             calling compute_totals on the problem.
+        show_progress : bool
```

Comment:
I think you should combine these two arguments into a single one.

The question to ask is, will these two arguments ever be used separately? If not, then there should be a way to combine. LIke before, something like "progress_stream", which is None by default.